src/System.Drawing.Common/src/System/Drawing/Bitmap.cs

```diff
@@ -8,11 +8,12 @@
 using System.IO;
 using System.Runtime.InteropServices;
 using System.Security.Permissions;
+using System.Runtime.Serialization;
 
 namespace System.Drawing
 {
     [ComVisible(true)]
-    public sealed partial class Bitmap : Image
+    public sealed partial class Bitmap : Image, ISerializable
```

Comment:
See comment above. Bitmap isn't implementing ISerializable by itself but by `System.Drawing.Image`. Remove it here.